VimeoNetworking.xcodeproj/project.pbxproj

```diff
@@ -2914,10 +2994,10 @@
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = NO;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_IDENTITY = "";
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
The `CODE_SIGN_IDENTITY` and `DEVELOPMENT_TEAM` should not be changed since this is a public repo.